sympy/printing/glsl.py

```diff
@@ -199,7 +198,7 @@ def _print_Function_with_args(self, func, func_args):
                 func = cond_func
             else:
                 for cond, func in cond_func:
-                    if cond(args):
```

Comment:
`args` not defined, Not sure how to reach that code, but will look more into it.

Reply:
Checking the code more carefully, I do not think it can be reached with the current code. It seems to be some sort of fallback in case the function is not already defined, or as some intermediate thing for future extensions.

First, the code checks if the function is in the list of known functions, if it is it gets the glsl name of the function. If that is not a string, it goes to the piece of code I changed. However, at the moment it is always a string.

Still, the updated code would not give an error when executed.